doc/rst/language/spec/statements.rst

```diff
@@ -927,6 +929,208 @@ only applicable to modules, please see :ref:`Using_Modules`.
 For more information on modules in general, please
 see :ref:`Chapter-Modules`.
 
+.. _The_Import_Statement:
+
+The Import Statement
+--------------------
+
+The import statement provides either only qualified access to all of the public
+symbols of a module or only unqualified access to the specified public symbols
+of a module.
+
+The syntax of the import statement is given by:
+
+.. code-block:: syntax
+
+   import-statement:
+     privacy-specifier[OPT] `import' module-or-symbol-rename ;
+     privacy-specifier[OPT] `import' module-or-symbol-base unqualified-list ;
+
+   module-or-symbol-rename:
+     rename-base
+     identifier . module-or-symbol-rename
```

Comment:
Can we chain renames?

Reply:
By chain, do you mean:
```chapel
module A {
  ...
}
module B {
  import A as Foo;
}
module C {
  import B.Foo as Bar;
}
```
?

Reply:
Not exactly. Reading this comment makes me think we can do
`import A.B.C.D.Foo as Bar;  // assuming we are fine visibility-wise`
Which makes sense that we should be able to do it, but still just gave me a short pause.